### PR TITLE
Update macOS platform view example code (swift)

### PR DESCRIPTION
**Pull request by fengjiongmax**
_Description of what this PR is changing or adding, and why:_
The original macOS platform view code has several syntax errors, and the private `NSView` was passed to the `createNativeView` function, not the `NativeView` itself, so the function will make no changes to the UI.

This PR fixes those errors, and removes the private `NSView`.

_Issues fixed by this PR (if any):_

_PRs or commits this PR depends on (if any):_

## Presubmit checklist

- [ ] This PR is marked as draft with an explanation if not meant to land until a future stable release.
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Comment by jmagman** [comment]
@cbracken can you take a look?

**Comment by sfshaza2** [comment]
@fengjiongmax, can you sign the CLA? Once approved, we can then land it.

**Comment by fengjiongmax** [comment]
I just signed the CTA, and hit the rescan.

File: src/content/platform-integration/macos/platform-views.md [hunk 1]
```diff
@@ -118,29 +118,28 @@ class NativeViewFactory: NSObject, FlutterPlatformViewFactory {
...
-
+    
+    required init?(coder nsCoder: NSCoder) {
+        super.init(coder: nsCoder)
+    }
+    
   func createNativeView(view _view: NSView) {
-    _view.wantsLayer = true
-    _view.layer?.backgroundColor = NSColor.systemBlue.cgColor
-    _view.frame = CGRect(x: 0, y: 0, width: 200, height: 200)
```
Comment (cbracken) [inline src/content/platform-integration/macos/platform-views.md:138]: Why remove the backgroundColor? This is there for consistency across iOS/macOS example code. Similar with the textColor below.

**Review by cbracken — APPROVED** [review]
Thanks for catching! I thought I had landed a commit for the `FlutterPlatformView` -> `NSView` change, but seems like I didn't push to the remote before I left on holiday. Thanks for the fix -- this looks great!

LGTM stamp from a Japanese personal seal

File: src/content/platform-integration/macos/platform-views.md (hunk 1 above)
Comment (sfshaza2) [reply src/content/platform-integration/macos/platform-views.md:138]: @fengjiongmax, please make Chris' suggested changes. thx

Comment (fengjiongmax) [reply src/content/platform-integration/macos/platform-views.md:138]: They were moved to the `init` function, should I move them back here?

**Review by sfshaza2 — APPROVED** [review]
lgtm, as per @cbracken. thx!

**Pull request merged**